sites/test-env/pages/orders/[order_id].md

```diff
@@ -0,0 +1,5 @@
+---
+breadcrumb: "SELECT first_name || ' ' || last_name || '''s ' || order_month || ' Order' as breadcrumb FROM orders WHERE id = '{params.order_id}'"
```

Comment:
```suggestion
breadcrumb: "SELECT first_name || ' ' || last_name || '''s ' || order_month || ' Order' as breadcrumb FROM orders WHERE id = '${params.order_id}'"
```
does this need a `$`

Reply:
Fixed here [479242a](https://github.com/evidence-dev/evidence/pull/1964/commits/479242a712df2ee60b486c6718eaef94a50480c6)